simulation/src/test/resources/txt/uncommon_flakes(Strategy)[CONCURRENCY_LIMITER_PIN_UNTIL_ERROR].txt

```diff
@@ -0,0 +1,11 @@
+success=99.0%	client_mean=PT0.000001S    	server_cpu=PT0.01S        	client_received=10000/10000	server_resps=10000	codes={200=9900, 500=100}
```

Comment:
Can we avoid renaming these? It looks like the names are identical, but we've just added `(Strategy)` to the filename.

Reply:
If I don't do that, I need to make up a fake Strategy, so to go around that I just used the standard TestInfo#getDisplayName(). Felt like a good compromise

Reply:
Done